src/Renderer/c3DEngine.js

```diff
@@ -430,11 +430,15 @@ var unpack1K = function unpack1K(color, factor) {
  * @returns {int} uuid's node
  * */
 c3DEngine.prototype.screenCoordsToNodeId = function screenCoordsToNodeId(mouse) {
+    if (mouse === undefined)
+        { mouse = new THREE.Vector2(Math.floor(this.width / 2), Math.floor(this.height / 2)); }
+
     var camera = this.camera.camera3D;
 
     camera.updateMatrixWorld();
 
     var buffer = this.renderTobuffer(mouse.x, this.height - mouse.y, 1, 1, RendererConstant.ID);
+    this.renderScene();
```

Comment:
This fix may be useful, but I don't understand how it's related to the current commit?
(I don't need this change to be removed though)